smsjwplatform/defaultsettings.py

```diff
@@ -44,34 +44,34 @@
 
 """
 
-SMS_OAUTH2_CLIENT_ID = None
```

Comment:
These default settings hsould be moved to automationoauthclient or automationoauthdrf as appropriate